common/storage.py

```diff
@@ -0,0 +1,23 @@
+import boto
+import boto.s3.connection
+
+from django.conf import settings
+
+
+def get_s3_connection():
+    if settings.S3_ACCESS_KEY and settings.S3_SECRET_KEY and settings.S3_HOST:
+        return boto.connect_s3(
+            aws_access_key_id=settings.S3_ACCESS_KEY,
+            aws_secret_access_key=settings.S3_SECRET_KEY,
+            host=settings.S3_HOST,
+            is_secure=False,
+            calling_format=boto.s3.connection.OrdinaryCallingFormat())
+    return None
+
+
+def create_bucket(s3_connection):
```

Comment:
Use one function for this:
`
def get_or_create_bucket(s3_con):
 bucket = s3_con.get_bucket(settings.S3_BUCKET_NAME)
 if bucket is None:
   return s3_conn.create_bucket(..)
`

If s3_conn.create_bucket return current bucket if it already exist, just use create_bucket only